src/Pages/Index.cshtml

```diff
@@ -16,7 +16,7 @@
 }
 
 <div class="container index-Top">
-    <img width="350px" src="images/mainLogo.PNG" alt="" class="title-image mt-2">
+    <img src="images/mainLogo.PNG" class="title-image mt-2">
```

Comment:
Maybe in the future we could add alt text so that the website would become more accessible with people with visual impairments or for those who have bad internet connections

Reply:
Well it would be nice small touch, but since it is not as important for us now, maybe I'll do this later :)